github.go

```diff
@@ -69,27 +69,53 @@ func checkOpenPRs(ctx *context.Context, client *github.Client, owner string, rep
 	}
 
 	for _, openPullRequest := range openPullRequests {
+		initialDate, err := initialDate(ctx, client, owner, repo, openPullRequest)
+		if err != nil {
+			fmt.Printf("Error %s\n", err)
+			os.Exit(1)
+		}
+
 		currentPr := prType{
 			client:    client,
 			ctx:       ctx,
 			owner:     owner,
 			repo:      repo,
-			OpenSince: daysSincePRCreated(openPullRequest.CreatedAt),
+			daysSince: daysSince(initialDate),
```

Comment:
Maybe `inactiveDays` is a little better then `daysSince` because the first question I have since what?